From 32fd6d03cf84f7ba0222168712567e52542cef73 Mon Sep 17 00:00:00 2001
From: "Mishra, ManishX" <manishx.mishra@intel.com>
Date: Thu, 7 Oct 2021 17:47:11 +0530
Subject: [PATCH] [EHL] Support SKU3 and SKU4 GPU media omx

Add support for GPU id 0x4555 as present in EHL SKU 3/3A/4 B1 variant

Tracked-On: OAM-99679
Signed-off-by: Mishra, ManishX <manishx.mishra@intel.com>
---
 omx_utils/src/mfx_omx_dev_android.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/omx_utils/src/mfx_omx_dev_android.cpp b/omx_utils/src/mfx_omx_dev_android.cpp
index 4f25934..6373d81 100644
--- a/omx_utils/src/mfx_omx_dev_android.cpp
+++ b/omx_utils/src/mfx_omx_dev_android.cpp
@@ -55,6 +55,7 @@ const mfx_device_item listLegalDevIDs[] = {
     { 0x4551, MFX_HW_EHL},
     { 0x4569, MFX_HW_EHL},
     { 0x4571, MFX_HW_EHL},
+    { 0x4555, MFX_HW_EHL},
     /* BXT */
     { 0x0A84, MFX_HW_BXT},
     { 0x0A85, MFX_HW_BXT},
-- 
2.33.0

